cmake/onnxruntime_server.cmake

```diff
@@ -1,8 +1,49 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
-
 set(SERVER_APP_NAME "onnxruntime_server")
 
+set(gRPC_BUILD_TESTS OFF CACHE INTERNAL "Don't build tests")
+set(gRPC_GFLAGS_PROVIDER "" CACHE INTERNAL "Don't use gflags")
+set(gRPC_BENCHMARK_PROVIDER "" CACHE INTERNAL "Don't use benchmark")
+set(gRPC_ZLIB_PROVIDER "package" CACHE INTERNAL "Use preinstalled zlib library")
```

Comment:
It will find whatever zlib can be found with find_packages. I think the build will fail if it's incompatible, but there's no check in the CMAKE and I don't know the requirements.